src/lib/Util/ContentTypeFieldsExtractorInterface.php

```diff
@@ -14,6 +14,8 @@ interface ContentTypeFieldsExtractorInterface
      * @return list<int>
      *
      * @throws \Ibexa\Contracts\Core\Repository\Exceptions\NotFoundException
+     * @throws \Ibexa\AdminUi\Exception\FieldTypeExpressionParserException
+     * @throws \LogicException
```

Comment:
This cannot be declared for the interface. It has to be a different type of exception, if needed.
```suggestion
```